debezium-core/src/main/java/io/debezium/connector/common/BaseSourceTask.java

```diff
@@ -273,7 +279,11 @@ public void commitRecord(SourceRecord record) throws InterruptedException {
     }
 
     @Override
-    public void commit() throws InterruptedException {
+    public void commit() {
```

Comment:
Let's keep the throws InterruptedException here.
It is there in the parent class method, so better to keep with the overridden one too, to maintain the contract.

For `performCommit()` I don't see an InterruptedException being thrown anywhere, so can be skipped.